public/scripts/dependencies/appwrite.js

```diff
@@ -1831,7 +1831,7 @@
                         payload['cursorDirection'] = cursorDirection;
                     }
                     if (typeof orderAttributes !== 'undefined') {
-                        payload['orderAttributes'] = orderAttributes;
+                        payload['orderAttributes'] = orderAttributes.map((orderAttribute) => '`' + orderAttribute + '`');
```

Comment:
Hmm. I'm not sure if this is the best place to fix this problem because the API can still throw an error if it's made outside of the Appwrite console (in other SDKs). It would probably be best to fix this server side in the the utopia database package.